k8s_local/adservice.yaml

```diff
@@ -0,0 +1,78 @@
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: adservice
+  labels:
+    app: adservice
+    component: otel-demo-adservice
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: adservice
+      component: otel-demo-adservice
+  template:
+    metadata:
+      labels:
+        team: team-opentelemetry
+        service: adservice
+        app: adservice
+        component: otel-demo-adservice
+    spec:
+      containers:
+      - name: adservice
+        image:  otel/demo:v0.2.0-alpha-adservice
```

Comment:
I think the image tag version should be a variable, and keep up to date.

Reply:
Made this an env var that we set in `Makefile`. Let me know if there is a better way to achieve same.